src/crypto.ts

```diff
@@ -125,12 +125,7 @@ export function hashRegistryEntry(registryEntry: RegistryEntry, hashedDataKeyHex
     dataKeyBytes = hashDataKey(registryEntry.dataKey);
   }
 
-  let dataBytes;
-  if (typeof registryEntry.data === "string") {
-    dataBytes = encodeUtf8String(registryEntry.data);
-  } else {
-    dataBytes = encodePrefixedBytes(registryEntry.data);
-  }
+  const dataBytes = encodePrefixedBytes(registryEntry.data);
```

Comment:
Q: Just so I understand, are we treating binary and string data the same now or do we expect to only get binary data?

Reply:
For the registry, we're using binary. This was triggered by confusion between Skylinks as Base64 representation of binary vs string stored as binary on the registry.

Reply:
The entry data can only be bytes now, yeah.